docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -61,6 +62,16 @@ \subsection{Environment}
 \wss{Hardware and software}
 
 \section{Goals}
+\begin{itemize}
+\subsection{Work Orders}
+\item Shop owners can create, update, view, filter, search work orders. Any details such as estimated time, parts used, and costs related to the work order and showing the technican assigned to each order. Allow the shop owner to send work order receipts to the customer and request payment.
+\subsection{Quotes}
+\item Shop owners can give quotes to customer requests. The shop owners will be able to chat with the vehicle owners and respond to requests easily through the application.
+\item Vechicle owners can request a quote on their job. This will allow vehicle owners to easily get quotes from multiple mechanics and compare prices.
```

Comment:
```suggestion
\item Vehicle owners can request a quote on their job. This will allow vehicle owners to easily get quotes from multiple mechanics and compare prices.
```